Formula/hapi-fhir-cli.rb

```diff
@@ -0,0 +1,154 @@
+class HapiFhirCli < Formula
+  desc "Command-line interface for the HAPI FHIR library"
+  homepage "http://hapifhir.io/doc_cli.html"
+  url "https://github.com/jamesagnew/hapi-fhir/releases/download/v3.3.0/hapi-fhir-3.3.0-cli.tar.bz2"
+  sha256 "8342c78598edd9b6509fff0b9cb1de9b277b97f1537342124d1e78523d887d15"
+
+  def install
+    inreplace "hapi-fhir-cli", /SCRIPTDIR=(.*)/, "SCRIPTDIR=#{prefix}"
+    prefix.install "hapi-fhir-cli.jar"
+    bin.install "hapi-fhir-cli"
+  end
+
+  test do
+    (testpath/"TestResource.json").write <<~EOS
```

Comment:
Could we get this rest resource from the package, or from a URL (as a resource), rather than inlining it all in the formula?

Reply:
Fixed